src/Microsoft.Azure.WebJobs.Host/Bindings/Runtime/Binder.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Azure.WebJobs
     /// <summary>
     /// Class providing functionality for dynamically binding to WebJobs SDK attributes at runtime.
     /// </summary>
-    public class Binder : IBinder, IWatchable, IDisposable
+    public class Binder : IWatchable, IDisposable
```

Comment:
>public [](start = 4, length = 6)

Can we make Binder private and leave IBinder public? 
A public interface is less liability than a public class.